5.application_scopes.md

```diff
@@ -65,185 +65,18 @@ The spec defines the constituent parts of a scope.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `type` | `string` | Y | | Determines type of the application scope. The core types are defined [above](#core-application-scope-types) |
 | `allowComponentOverlap` | `bool` | Y | | Determines whether a component is allowed to be in multiple instances of this scope type simultaneously. When false, the runtime implementation MUST produce an error and stop deployment if an attempt is made to place a component into more than one instance of this scope type simultaneously. |
-| `parameters` | [`[]Parameter`](#parameter) | N | | The scope's configuration options. |
+| `definitionRef` | [`DefinitionRef`](#definitionRef) | Y | | Index of the scope definition. |
 
-The `type` field defines the type name for this type. Examples may be found in the [Core Application Scope Types](#core-application-scope-types) above. Extended scope types MUST NOT be in the `core.oam.dev` namespace.  However, they must follow the [group, version, kind notation](2.overview_and_terminology.md). For example: `scopes.example.com/v1.ExtendedNetworkScope`
+### DefinitionRef
 
-#### Parameter
+We use the same [DefinitionRef](3.workload.md#DefinitionRef) in workload.
 
-The parameters that a scope exposes to operators.
 
-| Attribute | Type | Required | Default Value | Description |
-|-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The parameter's name. Must be unique per scope. |
-| `description` | `string` | N | | A description of the parameter. |
-| `type` | `string` | Y | | The parameter's type (One of `string`, `boolean`, `number`). |
-| `required` | `boolean` | N |`false` | Whether a value _must_ be provided for the parameter. |
-| `default` | type indicated by `type` field | N | | The parameter's default value. |
+## Deployment
 
-The `name` field allows Unicode letters, numbers and `_` and `-`. The `description` field allows those characters plus whitespace and punctuation characters.
+Application scope instances are defined and deployed by their standalone way. Go to scopes folder to see [core scopes](scopes/core.md) and [extended scopes](scopes/extended.md) details.
```

Comment:
What does "Application scope instances are defined and deployed by their standalone way" mean?